cms/src/main/java/com/visitscotland/brxm/validator/SharedLinkExternalValidator.java

```diff
@@ -0,0 +1,47 @@
+package com.visitscotland.brxm.validator;
+
+import com.visitscotland.brxm.hippobeans.Article;
+import com.visitscotland.brxm.hippobeans.ArticleSection;
+import com.visitscotland.brxm.hippobeans.SharedLink;
+import org.onehippo.cms.services.validation.api.ValidationContext;
+import org.onehippo.cms.services.validation.api.ValidationContextException;
+import org.onehippo.cms.services.validation.api.Validator;
+import org.onehippo.cms.services.validation.api.Violation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.jcr.Node;
+import javax.jcr.NodeIterator;
+import javax.jcr.RepositoryException;
+import javax.jcr.Value;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * jcr:Name = visitscotland:shared-link-bsh-external
+ *
+ * @author jose.calcines
+ */
+public class SharedLinkExternalValidator implements Validator<Node> {
+
+    private static final Logger logger = LoggerFactory.getLogger(LinkValidator.class);
```

Comment:
Small point 😄 typically the convention in Java is that static constants should be capitalised!

```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(LinkValidator.class);
```

What do you think? 😄

Reply:
The eternal question about Logger being a constant, right 😄?

I would lean toward consistency here, so given that the rest of the codebase is using logger as lower case I would stick with that.

About the observation itself. I have been in both sides on the fence and technically the logger is not inmutable therefore it cannot be classified as pure constant that's why it is set in lowercase (If you look at the [documentation](https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names) you can see that they also classify it as not constant.

However, I see your point, I personally agree with it. I preffer to use upper case so I can denote the intention of inmutability.

I hope this made sense

Reply:
This was very valuable, thank you! 😁